src/types/api.d.ts

```diff
@@ -29,12 +29,14 @@ export interface DataPlaneOverviewParameters extends PaginationParameters {
   /**
    * **Example**: `?gateway=delegated`
    */
-  gateway?: 'builtin' | 'delegated' | boolean
+  gateway?: 'builtin' | 'delegated' | 'true' | 'false' | boolean
```

Comment:
Non blocker, kinda thinking out loud: I keep seeing little places with gateway = true/false as opposed to gateway = 'true'/'false' i.e. boolean vs string. I wonder if we should remove the boolean so its always a string. Don't think it's that important, but it would help with things like - "should this be 'true' or true? 🤔 ". wdyt?

Reply:
Right, it’s nicer to only deal with string and number values (or possibly arrays of strings or numbers) on the parameter level. Technically, this wouldn’t be a problem because whatever piece of code turns these parameters into query parameters into a URL would always convert booleans appropriately.

I’m gonna remove them as booleans.

Reply:
Done in https://github.com/kumahq/kuma-gui/pull/505/commits/a091db498913daaae358c4fb2f4730bff4aef68e.